package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "joi-extension-semver",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

```suggestion
  "version": "2.0.0",
```

Could you hold off one second on this part? I'll update it as part of prepping a release + changelog.